tests/particle_attributes/particle_attributes.xml

```diff
@@ -0,0 +1,133 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
You've added 3 tests which is good. But you need to explain somewhere what functionality is being tested in each. What is different between particle_attributes and particle_attributes_parallel and particle_attributes_zoltan?